core/jvm/src/main/scala/cats/effect/IORuntimeCompanionPlatform.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+import java.util.concurrent.{CountDownLatch, Executors, TimeUnit}
+import java.util.concurrent.atomic.AtomicInteger
+
+private[effect] abstract class IORuntimeCompanionPlatform { self: IORuntime.type =>
+  lazy val default: IORuntime = build._1
+
+  private[effect] def build: (IORuntime, () => Unit) = {
```

Comment:
We could add an optional `def shutdown(): Unit` to `IORuntime` instead but that seemed kind of messy.

Reply:
`def build()` :-)